problem_set_1.py

```diff
@@ -1,38 +1,56 @@
 # 1. Set the variable `given_name` to the string "Addison".
-
+given_name = "Addison"
 
 # ------------------------------------------------------------------------------
 # 2. You have 20 candies that you must divide equally among 6 people. How many candies will be left over?
 # Set variables for `candies`, `people`, `left_over` to make your tests pass.
-
+candies = 20
+people = 6
+left_over = candies % people
 
 # ------------------------------------------------------------------------------
 # 3. Create a function called `greeting` that returns "Hello, <name>!",
 # where <name> is the name given as an argument to the function.
+def greeting(name):
+    return "Hello, " + name + "!"
 
 
 # ------------------------------------------------------------------------------
 # 4. Create a function called `is_odd` that, given a number, will
 # return true if the number is odd and false if it is not. An odd number is a
 # number which, when divided by 2, has a remainder of 1 or -1.
-
+def is_odd(number):
+    if number % 2 == 1 or number % 2 == -1:
+        return True
+    else:
+        return False
 
```

Comment:
Nice! A fun exercise might be to refactor these using `try...except` https://docs.python.org/3/reference/compound_stmts.html#the-try-statement

Reply:
I *think* I did this right--it passed the tests at least!--but I admit I'm not at all clear on when try/except should be used instead of if/elif/else (particularly because the linter in VSCode is insisting "do not use bare 'except'", but I'm at a loss as to which exception I should be using)